addons/rating/security/ir.model.access.csv

```diff
@@ -1,5 +1,5 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
-access_rating_user,rating.rating.user,rating.model_rating_rating,base.group_user,1,1,1,0
+access_rating_user,rating.rating.user,rating.model_rating_rating,base.group_user,1,0,1,0
```

Comment:
It is not just for the project users, but for all internal users. So we need to be sure if it is correct or not.
I think it is normal to edit the rating in the case where there is an error. 🤔